portality/migrate/2966_add_oa_start_date_from_backup/add_oa_start_date_from_backup.py

```diff
@@ -0,0 +1,36 @@
+import csv
+from copy import deepcopy
+
+from portality.models import Journal
+from portality.settings import BASE_FILE_PATH
+
+if __name__ == "__main__":
+
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-o", "--out", help="output file path")
+    args = parser.parse_args()
+
+    if not args.out:
+        print("Please specify an output file path with the -o option")
+        parser.print_help()
+        exit()
+
+    with open(args.out, "w", encoding="utf-8") as f:
+        writer = csv.writer(f)
+        writer.writerow(['Journal ID', 'OA Start Date'])
+
+        f = open(BASE_FILE_PATH + "/migrate/2966_add_oa_start_date_from_backup/oa_start_out.csv")
```

Comment:
As noted elsewhere, this csv probably shouldn't be in the code, so this script would need modifying to take the location of the csv as an argument